.github/workflows/docker-build.yml

```
@@ -0,0 +1,237 @@
+name: Publish Docker Image
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v1.*
+
+env:
+  REPOSITORY: ${{ secrets.DOCKER_HUB_ORGS }}/fluentd-kubernetes-daemonset
+jobs:
+  amd64:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        context:
+          - docker-image/v1.16/debian-azureblob
+          - docker-image/v1.16/debian-cloudwatch
+          - docker-image/v1.16/debian-elasticsearch7
+          - docker-image/v1.16/debian-elasticsearch8
+          - docker-image/v1.16/debian-forward
+          - docker-image/v1.16/debian-gcs
+          - docker-image/v1.16/debian-graylog
+          - docker-image/v1.16/debian-kafka
+          - docker-image/v1.16/debian-kafka2
+          - docker-image/v1.16/debian-kinesis
+          - docker-image/v1.16/debian-logentries
+          - docker-image/v1.16/debian-loggly
+          - docker-image/v1.16/debian-logzio
+          - docker-image/v1.16/debian-opensearch
+          - docker-image/v1.16/debian-papertrail
+          - docker-image/v1.16/debian-s3
+          - docker-image/v1.16/debian-syslog
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          platforms: linux/amd64
+      - uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      - name: Setup tags
+        run: |
+          set -x
+          context=${{ matrix.context }}
+          component=${context##*/}
+          for target in $(make echo-all-images); do
+            case $target in
+              *$component-amd64*)
+                tags=$(echo $target | cut -d':' -f2-)
+                tag1=$(echo $tags | cut -d',' -f1)
+                tag2=$(echo $tags | cut -d',' -f2)
+                echo "AMD64TAGS=${{ env.REPOSITORY }}:${tag1},${{ env.REPOSITORY }}:${tag2}" >> ${GITHUB_ENV}
+                ;;
+            esac
+          done
+      - name: Build and push for amd64
+        uses: docker/build-push-action@v6
+        with:
+          context: ${{ matrix.context }}
+          provenance: false
+          push: true
+          platforms: linux/amd64
+          tags: ${{ env.AMD64TAGS }}
+          # dare to use old mediatype (application/vnd.docker.distribution.manifest.v2+json)
+          outputs: oci-mediatypes=false
+      - name: Build and push for arm64
+        uses: docker/build-push-action@v6
+        with:
+          context: ${{ matrix.context }}
+          provenance: false
+          push: true
+          platforms: linux/amd64
+          tags: ${{ env.AMD64TAGS }}
+          # dare to use old mediatype (application/vnd.docker.distribution.manifest.v2+json)
+          outputs: oci-mediatypes=false
```

Looks like forgotten to remove the step for `arm64` in the `amd64` job.

```suggestion
      - name: Build and push for amd64
        uses: docker/build-push-action@v6
        with:
          context: ${{ matrix.context }}
          provenance: false
          push: true
          platforms: linux/amd64
          tags: ${{ env.AMD64TAGS }}
          # dare to use old mediatype (application/vnd.docker.distribution.manifest.v2+json)
          outputs: oci-mediatypes=false
```

---

Thanks!